src/midi/midi_fluidsynth.cpp

```diff
@@ -476,12 +476,23 @@ MIDI_RC MidiHandlerFluidsynth::ListAll(Program *caller)
 	auto *section = static_cast<Section_prop *>(control->GetSection("fluidsynth"));
 	const auto sf_spec = parse_sf_pref(section->Get_string("soundfont"), 100);
 	const auto sf_name = std::get<std::string>(sf_spec);
+	const size_t term_width = real_readw(BIOSMEM_SEG, BIOSMEM_NB_COLS);
+
+	auto write_line = [caller](bool highlight, const std::string &line) {
+		const char color[] = "\033[32;1m";
+		const char nocolor[] = "\033[0m";
+		if (highlight)
+			caller->WriteOut("* %s%s%s\n", color, line.c_str(), nocolor);
+		else
+			caller->WriteOut("  %s\n", line.c_str());
+	};
 
 	// If selected soundfont exists in the current working directory,
 	// then print it.
 	const std::string sf_path = CROSS_ResolveHome(sf_name);
-	if (path_exists(sf_path))
-		caller->WriteOut("  %s\n", sf_path.c_str());
+	if (path_exists(sf_path)) {
+		write_line((sf_path == selected_font), sf_name);
+	}
```

Comment:
- This write highlights regardless if FluidSynth is (or isn't) open
- The bottom write highlights only if FluidSynth `is_open`

I'm probably missing something - just wanted to flag it a double-check :slightly_smiling_face:

Reply:
This piece of code is here to display the font if sf2 file was found in path relative to current working dir.

I don't want to recurse into subdirectories of working dir looking for sf2 files, so this line is displayed only when **selected** font name is found and FluidSynth is active. When it's not active, selected font is set to empty string, therefore no fonts from working dir will be displayed.

Not the best logic, but I'm not sure how to handle fonts found via relative paths (e.g. the path can be `../other-dir/file.sf2`).